lib/grandpa/message_tracker_test.go

```diff
@@ -180,3 +181,50 @@ func TestMessageTracker_MapInsideMap(t *testing.T) {
 	_, ok = voteMsgs[authorityID]
 	require.True(t, ok)
 }
+
+func TestMessageTracker_handleTick(t *testing.T) {
+	kr, err := keystore.NewEd25519Keyring()
+	require.NoError(t, err)
+
+	gs, in, _, _ := setupGrandpa(t, kr.Bob().(*ed25519.Keypair))
+	gs.tracker = newTracker(gs.blockState, gs.messageHandler)
+
+	msg := &VoteMessage{
+		Round: 100,
+	}
+	gs.tracker.addVote(&networkVoteMessage{
+		msg: msg,
+	})
+
+	gs.tracker.handleTick()
+
+	select {
+	case v := <-in:
+		require.Equal(t, msg, v.msg)
+	case <-time.After(testTimeout):
+		t.Errorf("did not receive vote message")
+	}
+
+	// shouldn't be deleted as round in message >= grandpa round
+	require.Equal(t, 1, len(gs.tracker.voteMessages[common.Hash{}]))
```

Comment:
nit can we use a non-empty `common.Hash` as the key to make sure it's passed/set correctly in the production code? Same for below.

Reply:
updated